packages/react/src/auto/AutoTable.tsx

```diff
@@ -24,4 +24,5 @@ export type AutoTableProps<
   filter?: Options["filter"];
   actions?: TableOptions["actions"];
   excludeActions?: TableOptions["excludeActions"];
+  selectable?: boolean;
```

Comment:
This is making me realize that we're not exposing or passing along the Polaris table props -- this should have already been exposed via that channel I think.

Reply:
After our online discussion, I will update this PR to pass all the relevant Polaris index table props, namely

```
emptyState?
lastColumnSticky?
selectable?
hasZebraStriping?
resourceName?
{ singular: string; plural: string; }
condensed?
```